addon/components/ember-table.js

```diff
@@ -0,0 +1,564 @@
+import Ember from 'ember';
+import StyleBindingsMixin from 'ember-table/mixins/style-bindings';
+import ResizeHandlerMixin from 'ember-table/mixins/resize-handler';
+import RowArrayController from 'ember-table/controllers/row-array';
+import Row from 'ember-table/controllers/row';
+
+export default Ember.Component.extend(
+StyleBindingsMixin, ResizeHandlerMixin, {
+  classNames: ['ember-table-tables-container'],
+  classNameBindings: ['enableContentSelection:ember-table-content-selectable'],
+
+  // ---------------------------------------------------------------------------
+  // API - Inputs
+  // ---------------------------------------------------------------------------
```

Comment:
Unrelated question: when does the attrs hash land in Ember? Thanks to clearly documented distinctions between our public and private APIs we should be in a pretty good place to move to that once it's ready.